cli/src/complete.rs

```diff
@@ -230,7 +230,7 @@ pub fn aliases() -> Vec<CompletionCandidate> {
     })
 }
 
-fn revisions(revisions: Option<&str>) -> Vec<CompletionCandidate> {
+fn revisions(match_prefix: &str, filter: Option<&str>) -> Vec<CompletionCandidate> {
```

Comment:
nit: `filter` is ambiguous in this context. Maybe rename to `revset` or `revset_filter`?